static/app/views/performance/newTraceDetails/traceDrawer/details/missingInstrumentation.tsx

```diff
@@ -59,22 +64,39 @@ function EAPMissingInstrumentationNodeDetails({
 }) {
   const {node} = props;
   const previous = node.previous as TraceTreeNode<TraceTree.EAPSpan>;
-  const parentEAPTransaction = TraceTree.ParentEAPTransaction(previous);
 
-  const project = parentEAPTransaction
-    ? projects.find(proj => proj.slug === parentEAPTransaction.value.project_slug)
-    : undefined;
-  const profileId = parentEAPTransaction?.value.profile_id || '';
-  const profilerId = parentEAPTransaction?.value.profiler_id || '';
+  const {
+    data: eventTransaction,
+    isLoading: isEventTransactionLoading,
+    isError: isEventTransactionError,
+  } = useTransaction({
+    event_id: previous.value.transaction_id,
+    organization: props.organization,
+    project_slug: previous.value.project_slug,
+  });
+
+  const event = eventTransaction ?? null;
```

Comment:
nit: you should be able to do `data: eventTransaction = null` so you dont need to define another variable here